pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -94,8 +94,13 @@ buildPythonApplication rec {
     ./disable-test_ssh_bootstrap_with_different_launchers.patch
   ];
 
-  # Causes build failure due to warning
-  hardeningDisable = lib.optional stdenv.cc.isClang "strictoverflow";
+  hardeningDisable = [
+    # causes redefinition of _FORTIFY_SOURCE
+    "fortify3"
+  ] ++ lib.optionals stdenv.cc.isClang [
+    # Causes build failure due to warning
```

Comment:
Sounds like werror which can be turned off

Reply:
Thank you for prompting me on this. I'd like to explore fixing the warning entirely in https://github.com/NixOS/nixpkgs/pull/243595.